fineract-avro-schemas/src/main/avro/loan/v1/LoanInstallmentDelinquencyBucketDataV1.avsc

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "LoanInstallmentDelinquencyBucketDataV1",
+    "namespace": "org.apache.fineract.avro.loan.v1",
+    "type": "record",
+    "fields": [
+        {
+            "default": null,
+            "name": "delinquencyRange",
+            "type": [
+                "null",
+                "org.apache.fineract.avro.loan.v1.DelinquencyRangeDataV1"
```

Comment:
According to the design the fields here should have been moved up, like `installmentDelinquencyBuckets[n].minimumAgeDays` and yours is `installmentDelinquencyBuckets[n].delinquencyRange.minimumAgeDays` I think it's perfectly fine, but please update the documentation to make it inline with the implementation.

Reply:
done